AudioLink/Scripts/AudioLinkController.cs

```diff
@@ -64,7 +67,21 @@ void Update()
 
             void Start()
             {
-                if (audioLink == null) Debug.Log("Controller not connected to AudioLink");
+                if (audioLink == null) {
+                    Debug.LogError("Controller not connected to AudioLink");
+                    return;
+                }
+                if (themeColorController == null) {
+                    // This is here in case someone upgraded AudioLink, which updates
+                    // everything in the prefab, but not the outermost properties of the prefab.
+                    // TODO: Double check that this is how upgrading ends up working.
+                    Debug.Log("AudioLinkController using fallback method for finding themeColorController");
+                    Transform controllerTransform = transform.Find("ThemeColorController");
+                    themeColorController = (UdonBehaviour) controllerTransform.GetComponent(typeof(UdonBehaviour));
```

Comment:
This looks like a nullref exception waiting to happen in case "ThemeColorController" couldn't be found, or if it doesn't have an UdonBehaviour component. Perhaps add some null checks?

Reply:
Done. Added null-checks.